### PR TITLE
Translate hooks-intro.md

### PR DESCRIPTION
**Pull request by Jacco**
My first go at it, hope it goes well

Regards, Jacco

### CODE REVIEW

**Comment by degeens** [comment]
Thank you for contributing to the Dutch React docs! 🙏

I've been very busy lately, but I'll try to review your translations this week.

**Comment by Jacco** [comment]
@degeens that would be great 👍

File: content/docs/hooks-intro.md [hunk 1]
```diff
@@ -1,110 +1,110 @@
 ---
 id: hooks-intro
-title: Introducing Hooks
+title: Introductie tot Hooks
```
Comment (Jacco) [inline content/docs/hooks-intro.md]: Zwaait naar @degeens, volgens mij moet je hier kunnen reviewen

**Comment by degeens** [comment]
I'm closing this PR because PR #44 is replacing this one.

**Pull request closed without merging**